troubadix/plugins/overlong_description_lines.py

```diff
@@ -0,0 +1,91 @@
+#  Copyright (c) 2024 Greenbone AG
+#
+#  SPDX-License-Identifier: GPL-3.0-or-later
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from pathlib import Path
+from typing import Iterable, Iterator
+
+from troubadix.plugin import LineContentPlugin, LinterError, LinterResult
```

Comment:
I would suggest to use a `LinterWarning` instead of a `LinterError` as this is more of a coding style thing which shouldn't completely fail (at least not for now until all existing VTs have been adjusted accordingly which might take a good amount of time).

Reply:
implemented in 095e81bba6298a8cf4a30d61e1da71e18ecf9888